index.html

```diff
@@ -8,7 +8,7 @@
         <script>
             fetch('quoteFile.txt').then(response => response.text())
   .then(text => console.log(text))
-             document.write(text)
+             document.write(response.text)
```

Comment:
This isn't being added to the document.  But I do see the response in the console.

Reply:
Didn't intend to resolve in this pr.  Will resolve in a later PR.